api/app/events/tests.py

```diff
@@ -209,6 +209,16 @@ def test_get_events_unauthed_not_available(self):
         assert data[0]["description"] == 'Event Description'
         assert data[0]["status"] == 'Application not available'
 
+    def test_get_event_type(self):
+        self.seed_static_data()
+        db.session.delete(self.test_form)
+        db.session.commit()
+
+        response = self.app.get('/api/v1/events')
+        assert response.status_code == 200
```

Comment:
Please use `self.assertEqual(response.status_code, 200)` instead - this provides better error messages when the assertion fails (I know the rest of the file does it with Python assert, but we're slowly trying to change this :) )

Reply:
Bonus points if you'd like to clean up the rest of the asserts 🥇

Reply:
Ye Ill give it a go!